packages/flame/lib/src/components/position_component.dart

```diff
@@ -332,6 +337,15 @@ class PositionComponent extends Component
   /// The absolute center of the component.
   Vector2 get absoluteCenter => absolutePositionOfAnchor(Anchor.center);
 
+  /// Rotates the component to look at given target.
+  void lookAt(Vector2 target) {
```

Comment:
It's probably better to return the angle towards the target, instead of setting it directly. That way we can animate rotation towards the new direction instead of setting it directly.
```dart
double bearingOn(Vector2 target) => ...
```

Reply:
Also, might be good to document/test what happens when the target is on top of you.

Reply:
> double bearingOn(Vector2 target)

Are you also suggesting to rename `lookAt` to `bearingOn`?

As a non-native English speaker, it is not easy to guess that `bearingOn` has something to do with angles/orientation/rotation. Also, 3 of the common engines (Godot, Unreal and Unity) have a `lookAt` method with almost similar features. So keeping this method name similar to them will make it easier to find and understand.

Reply:
Perhaps "bearingOn" is too cryptic; how about `angleTo()`? The idea is that the `lookAt()` method would essentially be
```dart
void lookAt(Vector2 target) => angle = angleTo(target);
```

But most of the time you wouldn't want to "look at" something directly, but rather apply a rotation effect so that the component would turn to the new orientation smoothly, like
```dart
component.add(RotateTowardsEffect(component.angleTo(target), EffectController(speed: 1)));
```

Reply:
Yeah, `angleTo` sounds perfect.

Reply:
Added `angleTo` and modified `lookAt` to reuse it.